src/System.IO.Pipelines/System/IO/Pipelines/PreservedBuffer.cs

```diff
@@ -39,9 +39,6 @@ public void Dispose()
                     break;
                 }
             }
-
-            _buffer = default;
```

Comment:
Why undo this?

Reply:
It doesn't make sense. It's a struct there might be hundreds of copies cleaning one wouldn't magically return memory.